app/views/works/_meta.html.erb

```diff
@@ -22,8 +22,7 @@
           <dd class="<%= tag_type_css_class(type) %> tags">
             <ul class="commas">
               <% if (type == "ArchiveWarning" && hide_warnings?(work)) || (type == "Freeform" && hide_freeform?(work)) %>
-
-                <li id="work_<%= work.id %>_category_<%= type.tableize %>"><%= show_hidden_tags_link(work, type.tableize) %> </li>
```

Comment:
The old version of this code doesn't wrap the hidden tags link in `<strong>` tags, but I believe the new version does. So this will change the formatting slightly. Does the version number for the surrounding cache block need to be incremented, or is it a minor enough change that it's okay to have inconsistent formatting for a while?

Reply:
I've bumped it -- thanks, I didn't realize the toggle was inexplicably not bold on the meta before.